homeassistant/core.py

```diff
@@ -595,7 +595,7 @@ def async_add_hass_job(
                 hassjob.target = cast(
                     Callable[..., Coroutine[Any, Any, _R]], hassjob.target
                 )
-            task = self.loop.create_task(hassjob.target(*args), name=hassjob.name)
+            task = asyncio.create_task(hassjob.target(*args), name=hassjob.name)
```

Comment:
@bdraco is there reason to keep using the `loop.create_task` for performance reasons? If so I'll change this to just add comments why we do that.

Reply:
`asyncio.create_task` is great because you don't have to think about which loop to run the task in because it's always going to call `asyncio.get_running_loop()` for you, so it gets attached to the right loop.  We don't generally have that problem in HA since we only have one event loop.  So, I don't think we need the call to `asyncio.create_task` to solve that ambiguity.

We run ~1000-~2000 tasks per minute on the average system I see profiles for.  The cost difference will be the additional function overhead and calling `asyncio.get_running_loop()`

If we had a lot of mis-use of `async_create_task` being called in the executor, that would justify the cost of switching because it would mean calling `async_create_task` in the executor would now blow up with `RuntimeError: no running event loop` and protect us from a non thread safe operation.  If the CI blew up in lots of places I'd say the its worth the cost, but I don't think its a problem we generally have.

Reply:
Ok, I'll change to add a comment why we keep using `loop.create_task` in this place.

Reply:
https://github.com/home-assistant/core/pull/112372